subsys/power_seq/x86_non_dsx.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __X86_NON_DSX_H__
+#define __X86_NON_DSX_H__
+
+/* Delay in ms for pass through signals */
+#define POWER_EC_PCH_DSW_PWROK_DELAY_MS	100
+#define POWER_EC_PCH_RSMRST_DELAY_MS	10
+#define POWER_EC_PCH_SYS_PWROK_DELAY_MS	50
+#define POWER_EC_VR_EN_VCCIN_DELAY_MS	5
+#define POWER_EC_PCH_PM_PWRBTN_DELAY_MS	200
```

Comment:
should these be kconfig options?

Reply:
thank you for the feedback Jack.
These are constants needed by PCH, they don't change.